content/id/docs/concepts/workloads/controllers/garbage-collection.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Garbage Collection
+content_template: templates/concept
+weight: 60
+---
+
+{{% capture overview %}}
+
+Peran dari _garbage collector_ Kubernetes adalah untuk menghapus objek tertentu yang sebelumnya mempunyai pemilik, tetapi tidak lagi mempunyai pemilik.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Pemilik dan _dependent_
+
+Beberapa objek Kubernetes adalah pemilik dari objek lainnya. Sebagai contoh, sebuah ReplicaSet adalah pemilik dari sekumpulan Pod. Objek-objek yang dimiliki disebut _dependent_ dari objek pemilik. Setiap objek _dependent_ memiliki sebuah kolom `metadata.ownerReferences` yang menunjuk ke objek pemilik.
```

Comment:
nit: change _dependent_ -> dependen
change _independet_ -> independen
menunjuk -> merujuk

Reply:
fixed in https://github.com/kubernetes/website/pull/14513/commits/3842f1701da6c7406f2771419b0192e6ca5f7305